core-blocks/latest-comments/index.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Server-side rendering of the `core/latest-comments` block.
+ *
+ * @package gutenberg
+ */
+
+// Used to get `_draft_or_post_title`, which prevents empty-title posts from
+// appearing with no title.
+require_once( ABSPATH . 'wp-admin/includes/template.php' );
+
+$DEFAULT_COMMENTS_TO_SHOW = 5;
```

Comment:
This is not valid for phpcs, see: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions